util/src/table.rs

```diff
@@ -0,0 +1,122 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
```

Comment:
this should be split out into a separate PR and submitted separately. should also include testing.